test/libyul/yulSyntaxTests/number_literal_3.yul

```diff
@@ -1,3 +1,2 @@
 { let x:u256 := 1e5:u256 }
 // ----
-// ParserError 4828: (16-19): Invalid number literal.
```

Comment:
This changes the current behaviour for scientific notation literals, and is not in the scope of this task. Please revert.